input/pagecontent/5_content-sharing.md

```diff
@@ -0,0 +1,78 @@
+
+In order to avoid lost updates and other out of sync conditions, the Hub serves as the transaction coordinator.  It fulfills this responsibility by creating a version of the content's state with each update operation.  If an operation is requested by a client which provides an incorrect version, this request is rejected.  This approach is similar to the version concurrency approach used by [FHIR versions and managing resource contention](https://www.hl7.org/fhir/http.html#concurrency).  Additionally, many of the FHIRcast content sharing concepts have similarities to the [FHIR messaging mechanisms](https://www.hl7.org/fhir/messaging.html) and where possible the approaches and structures are aligned.
+
+FHIR resources are used to convey the structured information being exchanged in `[FHIR resource]-update` operations.  However, it is possible that these resources are never persisted in a FHIR server.  During the exchange of information, the content (FHIR resource instances) is often very dynamic in nature with a user creating, modifying, and even removing information which is being exchanged.  For example, a measurement made in an imaging application could be altered many times before it is finalized and it could be removed.
+
+### Responsibilities of a FHIRcast Hub and a Subscribed Client
+
+If supporting content sharing, a FHIRcast Hub MUST fulfill additional responsibilities:
+1. Assign and maintain an anchor context's `versionId` when processing a `[FHIR resource]-open` request - the `versionId` does not need to follow any convension but must unique in the scope of a topic  
+2. Validate `[FHIR resource]-update` and `[FHIR resource]-select` requests for the correct `versionId` and reject the request if the version does not match the current `versionId` by returning a 4xx/5xx HTTP Status Code rather than updating the content or indication of selection
+3. Assign and maintain a new `versionId` for the anchor context's content and provide the new `versionId` along with the `priorVersionId` in the event corresponding to the validated update request
+4. Maintain a list of current FHIR resource content in the anchor context so that it may provide the anchor context's content in response to a [`GET Context`](3-16-get-context.html) request
+5. When a `[FHIR resource]-close` request is received, the Hub should dispose of the content for the anchor context since the Hub has no responsibility to persist the content
+
+A Hub is not responsible for structurally validating FHIR resources.  While a Hub must be able to successfully parse FHIR resources sufficiently to perform its required capabilities (e.g., find the `id` of a resource), a Hub is not responsible for additional structural checking. 
+
+A Hub is not responsible for any long-term persistence of shared information and should purge the content when a `[FHIR resource]-close` request is received.
```

Comment:
I thought we'd identified scenarios when this won't work. Mediocre notes here: https://confluence.hl7.org/pages/viewpage.action?pageId=130485460

Reply:
In the notes I only see the comments/questions/concerns on the -select operation.  What other situations are you referring to?